4.0_Jedi_Training.py

```diff
@@ -2,53 +2,55 @@
 
   #1. Make the following program work. (3 mistakes)
      
-     midichlorians = float(input("Enter midichlorian count: ")
-     if midichlorians > 10000
-         print("You have serious Jedi potential")
-     elif:
-         print("Jedi, you will never be.")
+midichlorians = float(input("Enter midichlorian count: "))
+if midichlorians >10000:
+    print("You have serious Jedi potential")
+else:
+    print("Jedi, you will never be.")
 
 
  # 2. Make the following program work. (3 mistakes)
      
-     x = input("Enter a number: ")
-     if x = 3
-         print("You entered 3")
+x = int(input("Enter a number: "))
+if x == 3:
+    print("You entered 3")
+
 
 
   # 3. Make the following program work. (4 mistakes)
      
-     answer = input("What is the name of Poe Dameron's Droid? ")
-     if a = "BB8":
-         print("Correct!")
-         else
-         print("Incorrect! It is BB8.")
+answer = input("What is the name of Poe Dameron's Droid? ")
+if answer =="BB8":
+    print("Correct!")
+else:
+    print("Incorrect! It is BB8.")
 
 
   # 4. Make the following program work. (4 mistakes)
      
-     x = input("Name one of the top 3 greatest Jedi.")
-     if jedi == Yoda or Luke Skywalker or Obi-Wan Kenobi:
-         print "That is correct!"
+x = input("Name one of the top 3 greatest Jedi.")
+if x == "Yoda" or "Luke Skywalker" or "Obi-Wan Kenobi":
```

Comment:
This won't work. Needs to be if x.lower() == "yoda" or x.lower() == "luke skywalker"  etc.